core/src/main/java/com/adobe/aem/commons/assetshare/components/details/impl/MetadataImpl.java

```diff
@@ -152,7 +152,8 @@ public boolean isEmpty() {
         if (StringUtils.isBlank(getPropertyName())) {
             return true;
         } else {
-            return combinedProperties.get(getPropertyName()) == null;
+            Object val = combinedProperties.get(getPropertyName());
```

Comment:
computed properties can return "anything" - so should this check the type?

Also - i wonder if we should also check for instance off Array, or Collection (which are the 2 other super-common return types) and do null / empty checks on those?

```suggestion
            Object val = combinedProperties.get(getPropertyName());
            if (val == null ) { 
              return true; 
            } if (val instanceof String.class) {
                 return StringUtils.isBlank((String)val);
             } else {
                return false;
            }
```

Reply:
Agreed and updated.  Also added handling of an empty String[] property.